README.md

```diff
@@ -8,13 +8,18 @@
 
 ## Built With
 
-- HTML, CSS, JS
-- Bootstrap, SCSS, React, Redux
+- HTML, CSS, SCSS, JS
+-  React, Redux
+## Presentatin Link
 
+- [Video Demo](https://www.loom.com/share/e3fdfcaa2a2842a79baa3b318268fe62)
 
 ## Live Demo
 
-- [Live Demo Link]
+
+- [Live Demo Link](https://samiullahbahadur.github.io/Covid-tracker/)
```

Comment:
- [ ] Please kindly submit the link to the deployed site, this link takes you to your readme file. Please submit the link to your live site so others can see how your app is working. Make sure project is deployed properly, the home and detail page should work as expected.

Reply:
in the netlify and heroku site  I have an issue to deploy my project i tried many time unfortunately can't solve it